### ISSUE TITLE
Date field with display type set to Dropdowns fails

### ISSUE THREAD

**Issue by JeanLucEsser** [verbb/formie#978]
### Describe the bug

Adding a Date field with its Display Type set to Dropdows generates an error on loading the form:

```
Object of class craft\fields\data\SingleOptionFieldData could not be converted to int
```

### Steps to reproduce

1. Create a form with a date field with display type set to Dropdows
2. Try rendering the form

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.1.3

### Plugin version

2.0.0

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Just going to keep these open in case people have similar issues!

**Comment by engram-design** [comment]
Fixed in [2.0.1](https://github.com/verbb/formie/releases/tag/2.0.1)
